### ISSUE TITLE
[Bug Report] $vuetify.application.top wrong on SSR which lead to floating content on page load

### ISSUE THREAD

**Issue by aldarund** [vuetifyjs/vuetify#4190]
### Versions and Environment
**Vuetify:** 1.1.0-beta.1
**Vue:** 2.5.16
**Browsers:** Chrome 67.0.3396.62
**OS:** Windows 10

### Steps to reproduce
 $vuetify.application.top always stay to 56 on SSR no matter what once it is set somewhere, which lead to rendering glitches. And  for example on page with actual toolbar it is set to 56 on server, but then rerendered on client to 64 which also same  visual glitch\flashing.
Here is reproducible example
https://glitch.com/edit/#!/judicious-lemonade
https://judicious-lemonade.glitch.me/
If change something in code so it recompile and then load index page - all fine. Even if go links all is fine. But if reload on toolbar so ssr will render it, and then reload on home page it will add padding-top and will have that floating link

Basically on first page load e.g. index without toolbar ssr and client top is same  == 0. If you load second page via SSR ( f5 reload) it will set top ssr to be 56 and now content on first page will float on every f5 load of home page. And even on toolbar page client top is 64 while ssr still return 56 which lead to small flashing of toolbar.

### Expected Behavior
ssr and client padding same so there no floating\flashing content

### Actual Behavior
ssr and client padding different so there floating\flashing content

### Reproduction Link
[https://github.com/aldarund/vuetify-ssr-top/tree/glitch](https://github.com/aldarund/vuetify-ssr-top/tree/glitch)

Any workaround for this?

**Comment by johnleider** [comment]
This is still something we are actively looking for a fix but unfortunately did not make the cut for v2.0.  Thank you for your patience.

**Comment by KaelWD** [comment]
> ssr and client padding same so there no floating\flashing content

We don't have any plans to change this behaviour, but the SSR mismatch was fixed with #3436
